creator/decorators.py

```diff
@@ -26,9 +28,14 @@ class task:
     """
     A decorator to uniformly setup tasks that get enqueued for workers to
     execute.
+
+    If the wrapped task contains a release_id or task_id, save the log to the
+    appropriate model so that it may be retrieved relationally.
     """
 
     def __init__(self, job=None):
+        self._release = None
```

Comment:
I think you may be able to generalize the task decorator a bit. It looks like this method does the same thing for both Release and ReleaseTask (and later will also include IngestRun) - attach a JobLog to the object. 

So rather than passing in the foreign key parameter for each model class, could you have one keyword arg called `attach_log_to` (or something else?) which would take in a list of tuples: `[(Release, <the release id>), (ReleaseTask, <the release task id>)]` and then attach JobLogs to each model class instance if it exists (so basically what you do now for release and task)